src/improvements/tasks/sep/024-opcm-upgrade-superchainconfig-v410-freya-bn-u15/.env

```diff
@@ -0,0 +1 @@
+TENDERLY_GAS=32000000
```

Comment:
This is high, is that fine?

Reply:
It's the gas that we used for U16, to be honest I just copied it. I don't think it has any side effect, I could lower it if needed but I don't think anything will change